react/features/notifications/actions.ts

```diff
@@ -116,12 +116,20 @@ export function showNotification(props: INotificationProps = {}, type?: string)
         const { disabledNotifications = [], notifications, notificationTimeouts } = getState()['features/base/config'];
         const enabledFlag = getFeatureFlag(getState(), NOTIFICATIONS_ENABLED, true);
 
+        const { descriptionKey, titleKey } = props;
+
         const shouldDisplay = enabledFlag
-            && !(disabledNotifications.includes(props.descriptionKey ?? '')
-                || disabledNotifications.includes(props.titleKey ?? ''))
+            && !(disabledNotifications.includes(descriptionKey ?? '')
+                || disabledNotifications.includes(titleKey ?? ''))
             && (!notifications
-                || notifications.includes(props.descriptionKey ?? '')
-                || notifications.includes(props.titleKey ?? ''));
+                || notifications.includes(descriptionKey ?? '')
+                || notifications.includes(titleKey ?? ''));
+
+        if (typeof APP !== 'undefined') {
```

Comment:
shouldn't it be sent only when shouldDisplay is true? Or add shouldDisplay on the API notification payload?

Reply:
no, we always want the event to be triggered, leaving it up to the consumer to handle it whichever way they want